src/forcing/ForcingsEngineDataProvider.cpp

```diff
@@ -0,0 +1,48 @@
+#include <forcing/ForcingsEngineDataProvider.hpp>
```

Comment:
I think this file should be guarded by `NGEN_WITH_PYTHON`

Reply:
Just to clarify, you mean `ForcingsEngineDataProvider.hpp` should have a `NGEN_WITH_PYTHON` guard inside of it?

Reply:
Anything that requires python should be